source/Reloaded.Mod.Launcher.Lib/Models/ViewModel/Dialog/InstallPackageViewModel.cs

```diff
@@ -0,0 +1,65 @@
+namespace Reloaded.Mod.Launcher.Lib.Models.ViewModel.Dialog;
+
+/// <summary>
+/// ViewModel for downloading an individual package.
+/// </summary>
+public class InstallPackageViewModel : INotifyPropertyChanged
```

Comment:
You should use autogeneration for INotifyPropertyChanged here.
It's been a while since I've worked on R2 but I believe this uses Fody.PropertyChanged.